src/tests/ftest/erasurecode/restart.py

```diff
@@ -54,8 +54,10 @@ def execution(self, agg_check=None):
         self.get_dmg_command().system_start()
 
         if agg_check == "After":
+            size_after_restart = self.pool.pool_percentage_used()
+            self.log.info("Size after Restarti: %s ", self.pool.pool_percentage_used())
```

Comment:
Is there a reason we are querying the pool again by calling `self.pool.pool_percentage_used()` vs. displaying what was queried in the line before?
```suggestion
            self.log.info("Size after Restart: %s ", size_after_restart)
```

Reply:
size_after_restart is used for comparison later.